components/fieldgroups/experience-event/experienceevent-channel-referrer.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/experienceevent-channel-referral",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Channel Referral Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Channel referral information such as the referring source.",
+  "definitions": {
+    "experienceevent-channel-referral": {
+      "properties": {
+        "xdm:channel": {
```

Comment:
There is already a field named "xdm:channel" in the "Channel Details" field group that is intended to extend experience event and this will collide. Could you use a different name for this field?

Reply:
I intended for it to be the same channel structure and just add a new field under it.  channel is still an object and has the sane name and description.  shouldn't they merge together if you use both field groupd?

Reply:
As discussed offline let's move this field to the existing "Channel Details" fieldgroup.